pallets/gear/src/runtime_api.rs

```diff
@@ -69,16 +72,45 @@ where
 
         QueueOf::<T>::clear();
 
+        let map_extrinsic_err = |extrinsic_name: &'static str,
+                                 e: DispatchErrorWithPostInfo<PostDispatchInfo>|
+         -> Vec<u8> {
+            let error_module_idx = if let DispatchError::Module(ModuleError { index, .. }) = e.error
+            {
+                Some(index as usize)
+            } else {
+                None
+            };
+
+            let error_message: &'static str = e.into();
+
+            let gear_module_idx =
+                <<T as frame_system::Config>::PalletInfo as PalletInfo>::index::<Pallet<T>>()
```

Comment:
Might be replaced with a type alias.
```suggestion
                PalletIdexOf<T>()
```